app/controllers/candidate_interface/degrees/type_controller.rb

```diff
@@ -43,15 +43,15 @@ def update
     private
 
       def set_degree_type_names
-        @degree_types = Hesa::DegreeType.abbreviations_and_names.sort
+        @degree_types = Hesa::DegreeType.all
       end
 
       def conditionally_render_new_degree_type_form
         if degree_already_added?
-          @degree_types = Hesa::DegreeType.abbreviations_and_names
+          set_degree_type_names
```

Comment:
Minor: would prefer to set the ivar inline here, as in the alternative branch of the if?